test/config-next/va-remote-a.json

```diff
@@ -34,7 +38,6 @@
   },
 
   "common": {
-    "dnsResolver": "127.0.0.1:8053",
```

Comment:
Why did this get hoisted out of "common"? Should `dnsTimeout` and `dnsAllowLoopbackAddresses` be moved as well?

Reply:
Hoisting `dnsResolver` was part of https://github.com/letsencrypt/boulder/pull/3612, that PR just missed the remote VAs.

Reply:
Gotcha. thanks!